android/java/org/chromium/chrome/browser/crypto_wallet/util/AndroidUtils.java

```diff
@@ -12,4 +15,21 @@ public static int getToolBarHeight(Context context) {
         }
         return 0;
     }
+
+    public static void disableViewsByIds(View view, int... ids) {
+        if (view != null) {
+            for (int id : ids) {
+                disableView(view, id);
+            }
+        }
+    }
+
+    public static void disableView(View view, @IdRes int id) {
+        if (view == null) return;
+        View mView = view.findViewById(id);
```

Comment:
nit: it is fine for this method but avoid using member variable naming signature for local variable